tests/test_misc.lua

```diff
@@ -1,7 +1,7 @@
 local helpers = dofile('tests/helpers.lua')
 
 local child = helpers.new_child_neovim()
-local expect, eq = helpers.expect, helpers.expect.equality
+local expect, eq, noeq = helpers.expect, helpers.expect.equality, helpers.expect.no_equality
```

Comment:
If you use it once, no need to make a helper.

Reply:
Fixed.